Core/UserDefaultsPropertyWrapper.swift

```diff
@@ -84,9 +84,9 @@ public struct UserDefaultsWrapper<T> {
                 "com.duckduckgo.ios.autofillCredentialsHasBeenEnabledAutomaticallyIfNecessary"
         case autofillImportViaSyncStart = "com.duckduckgo.ios.autofillImportViaSyncStart"
         case autofillSurveyEnabled = "com.duckduckgo.ios.autofillSurveyEnabled"
-        case autofillSearchDauDate = "com.duckduckgo.ios.autofillSearchDauDate"
-        case autofillFillDate = "com.duckduckgo.ios.autofillFillDate"
-        case autofillOnboardedUser = "com.duckduckgo.ios.autofillOnboardedUser"
+        case autofillSearchDauDate = "com.duckduckgo.app.autofill.SearchDauDate"
```

Comment:
I'm planning to move AutofillPixelReporter to BSK (or as much as possible) so updating this now to a BSK friendly name to save handling a migration later